guides/Features.md

```diff
@@ -106,7 +107,7 @@ try {
     val result = yChat.completion()
         .setInput("Say this is a test.")
         .setMaxTokens(1024)
-        .set... // you can set more parameters
+        .set // you can set more parameters
```

Comment:
Can we provide an example of specific parameters?

Reply:
Sure! I have specified all parameters